SPECS/ca-certificates/cert2certdata.sh

```diff
@@ -6,10 +6,11 @@ set -e
 
 if [[ $# -lt 2 ]]; then
     echo "Usage: $0 <CA certificate in DER/PEM format> <trust flags in the format accepted by the '-t' argument for 'nss-addbuiltin'>" >&2
```

Comment:
nit: If I'm understanding this is operating on a directory now, might want to update the usage to match.